src/System.Runtime/ref/project.assets.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Of course we need to find a better place for this file, I just didn't know which one to use yet.